OpenFK/OFK.Common/RDFManager.cs

```diff
@@ -60,8 +58,8 @@ public static string DecodeRDF(string encoded)
                 {
                     if (col[currentCol][i] == character)
                     {
-                        ret[pos++] = code[i];
-                        if (code[i] == (char)0x3E)
+                        ret[pos++] = (char)i;
+                        if (i == (char)0x3E)
```

Comment:
I think you're trying to compare two different types here. Int on the left, char on the right. I'm not sure whether C# cares about that, but by removing the cast you get less code (yes I'm nitpicking lol).
```suggestion
                        if (i == 0x3E)
```

Reply:
Ah yeah my bad, I believe that cast was leftover from when I was trying to use a byte or char as the index. Tested without it just to be extra sure and it works.

Reply:
Awesome!